canonical-web-and-design/ubuntu.com
# Add redirect for /engage/charmed-openstack-adoption-whitepaper

## Done

- Add redirect for /engage/charmed-openstack-adoption-whitepaper

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/charmed-openstack-adoption-whitepaper
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to http://0.0.0.0:8001/engage/openstack-deployment-guide

## Issue / Card

Fixes #8708
